hbase-checkstyle/src/main/resources/hbase/checkstyle-suppressions.xml

```diff
@@ -37,6 +37,7 @@
   <suppress checks="VisibilityModifier" files=".*/src/test/.*\.java"/>
   <suppress checks="InterfaceIsTypeCheck" files=".*/src/main/.*\.java"/>
   <suppress checks="EmptyBlockCheck" files="TBoundedThreadPoolServer.java"/>
+  <suppress checks="EmptyBlockCheck" files=".*/src/test/.*\.java"/>
```

Comment:
Maybe too general? We disable this check in our the tests?

Reply:
Yes, you're right. That's too general. Will change it.